src/Compilers/Core/Portable/Syntax/GreenNode.cs

```diff
@@ -39,18 +39,7 @@ private string GetDebuggerDisplay()
         private NodeFlagsAndSlotCount _nodeFlagsAndSlotCount;
         private int _fullWidth;
 
-        protected NodeFlags flags
-        {
-            get => _nodeFlagsAndSlotCount.NodeFlags;
-            set => _nodeFlagsAndSlotCount.NodeFlags = value;
-        }
-
-        /// <inheritdoc cref="NodeFlagsAndSlotCount.SmallSlotCount"/>>
-        private byte _slotCount
```

Comment:
since this was private, i just inlined it into the tiny handful of cases in this code that referenced it.